templates/customize-class.html

```diff
@@ -1,92 +1,46 @@
-                                    {# Note: this code is copy/pasted in teachers.ts. If you change it here, also change it there #}
-                                    <div draggable="true" class="tab {% if adventure['from_teacher'] %}teacher_tab{% endif %} z-10 whitespace-nowrap flex items-center justify-left relative" tabindex="0" adventure="{{ adventure['name'] }}" level="{{ level }}" from-teacher = "{% if adventure['from_teacher'] %}true{% else %}false{% endif %}">
-                                        <span class="absolute top-0.5 right-0.5 text-gray-600 hover:text-red-400 fa-regular fa-circle-xmark" data-cy="hide"></span>
-                                        {{ adventure_names[adventure['name']] }}
-                                    </div>
-                                {% endif %}
-                            {% endfor %}
-                        </div>
+                    <option id="select-{{i}}" value="{{ i }}" {% if i == min_level %} selected {% endif %}>
+                        {{ _('level_title') }} {{ i }}
+                    </option>
                     {% endfor %}
-                {% else %}
-                    {% for level, adventures in adventures_default_order.items() %}
-                        <div id="level-{{ level }}" data-cy="level-{{ level }}" {% if level|int == 1 %}
-                                                                                    class="adventures-tab flex"
-                                                                                {% else %}
-                                                                                    class="adventures-tab hidden"
-                                                                                {% endif %}
-                                                                                >
-                            {% for adventure in adventures %}
-                            <div draggable="true" class="tab z-10 whitespace-nowrap flex items-center justify-left relative" tabindex="0" adventure="{{ adventure }}" level = "{{ level }}" from-teacher = "false">
-                                <span id="remove" class="absolute top-0.5 right-0.5 text-gray-600 hover:text-red-400 fa-regular fa-circle-xmark" data-cy="hide"></span>
-                                {{ adventure_names[adventure] }}
-                            </div>
-                            {% endfor %}
-                        </div>
-                    {% endfor %}
-                {% endif %}
-            </div>
-            <div class="flex flex-row justify-center items-center ml-2 w-2/12 mr-4">
-                <select class="py-2 px-1 w-full mt-2 h-10 text-center" name="levels" id="available" data-cy="available_adventures_current_level">
-                    <option value="none" selected>{{_('available_adventures_level')}} -</option>
                 </select>
             </div>
+            {{ render_partial('customize-class/sortable-adventures.html', 
```

Comment:
Ah, there it is! Can you explain to me what this does differently from the built-in `render_template`?